Sources/TuistCore/Utils/RootDirectoryLocator.swift

```diff
@@ -32,7 +32,7 @@ public final class RootDirectoryLocator: RootDirectoryLocating {
         } else if fileHandler.exists(path.appending(RelativePath(Constants.tuistDirectoryName))) {
             cache(rootDirectory: path, for: source)
             return path
-        } else if fileHandler.exists(path.appending(RelativePath(".git"))) {
+        } else if fileHandler.directoryExists(path.appending(RelativePath(".git"))) {
```

Comment:
there's a method `isFolder` on `FileHandler` already, that can be combined with the check. There isn't a need for the additional `FileHandler` methods.

Reply:
Ah, I missed that. Fixed.